CRM/Core/BAO/ActionSchedule.php

```diff
@@ -615,15 +615,15 @@ protected static function sendReminderEmail($tokenRow, $schedule, $toContactID):
     $mailParams = [
       'groupName' => 'Scheduled Reminder Sender',
       'from' => self::pickFromEmail($schedule),
-      'toName' => $tokenRow->context['contact']['display_name'],
```

Comment:
This code was taking advantage of the fact the token compat subscriber was loading the full contact - I have switched to a 'pseudotoken' approach to 'direct' it to

Reply:
Ah, that's clever to tell it to fetch `display_name`. 👍

Reply:
I'm glad you think it's clever ....